general/jobSubmission/initiateDataFolders.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# create default data directories in given location
+
+set -e
+
+# one argument: directories location
+
+cd $1
```

Comment:
Minor: Could you add quotations around $1 to prevent splitting. This command wouldn't fail, but give undefined behaviour if a folder is entered that has a space in it.